content/en/docs/tasks/debug-application-cluster/debug-running-pod.md

```diff
@@ -86,7 +87,7 @@ Ephemeral Containers:
     Mounts:         <none>
 ```
 
-* You can use arbitrary container image using `kubectl debug --image=$IMAGE_NAME`.
+* You can use an arbitrary container image using `kubectl debug --image=$IMAGE_NAME`.
```

Comment:
nit: remove `using`: ... use an arbitrary container image, for example, `kubectl debug ...`.